js/modules/popupRegistration.js

```diff
@@ -0,0 +1,76 @@
+const body = document.body;
+const popupLinks = document.body.querySelectorAll(".popup-link");
```

Comment:
use just document, not document.body
search through dom won't be slower but string of code will be shorter and clearer

Reply:
Dome.